packages/skeleton/src/lib/components/Paginator/Paginator.svelte

```diff
@@ -1,7 +1,8 @@
 <script lang="ts">
 	import { createEventDispatcher } from 'svelte';
 	// Types
-	import type { CssClasses, PaginationSettings } from '../../index.js';
+	import type { PaginationSettings } from '../../index.js';
```

Comment:
While not relevant to this PR, might as well take the opportunity to fix this import.
```suggestion
	import type { PaginationSettings } from './types.js';
```